set.mm

```diff
@@ -494421,6 +494469,67 @@ such that every prime ideal contains a prime element (this is a
         AXSXTNECGXDKXKVKVMVFVNVOVPWEWGWMVQVLVRVSBWHFGWLWDIWFLMWFTWHTPWLTULWDBHV
         TWAWB $.
     $}
+
+    ressply1.1 $e |- P = ( S |`s B ) $.
+    ${
+      $d B y $.  $d P y $.  $d U y $.  $d X y $.  $d ph y $.
+      ressply1invg.1 $e |- ( ph -> X e. B ) $.
+      $( A restricted polynomial algebra has the same group inverse.
+         (Contributed by Thierry Arnoux, 30-Jan-2025.) $)
+      ressply1invg $p |- ( ph -> ( ( invg ` U ) ` X ) = ( ( invg ` P ) ` X ) )
```

Comment:
"A restricted polynomial algebra" -> "An element of a restricted polynomial algebra"

Reply:
Done
(I guess I was thinking "same group inverse _operation_")